From: Richard Hansen <rhansen@rhansen.org>
Date: Sat, 23 May 2020 16:21:16 -0400
Subject: Rename RELEASENOTE to NEWS

This ensures that the file is installed as NEWS.gz as required in
Debian policy >= 4.2.0. For details, see:
https://www.debian.org/doc/debian-policy/ch-docs.html#changelog-files-and-release-notes

Forwarded: no
---
 RELEASENOTE => NEWS | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename RELEASENOTE => NEWS (100%)

diff --git a/RELEASENOTE b/NEWS
similarity index 100%
rename from RELEASENOTE
rename to NEWS
